extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -251,10 +243,12 @@ export class AmpAd3PImpl extends AMP.BaseElement {
         this.iframe_ = getIframe(this.element.ownerDocument.defaultView,
             this.element);
         this.apiHandler_ = new AmpAdApiHandler(
-          this, this.element, this.boundNoContentHandler_);
+            this, this.element, this.boundNoContentHandler_);
         return this.apiHandler_.startUp(this.iframe_, true);
       });
     }
+    dev().error('AMP_AD',
```

Comment:
I thought ad layoutCallback should always create a new iframe. what are cases where we schedule layout for `amp-ad` with an existing iframe?